packages/infobox/README.md

```diff
@@ -0,0 +1,5 @@
+# InfoBox Name
+
+InfoBox documentation
+
+<!-- TODO: collaborate with Design team to add documentation from Confluence doc -->
```

Comment:
@lilysamimi - should I just create a follow-up ticket for documentation? We can pair like we did for the Table docs :)